tasks/tasks/mixmasta_processors.py

```diff
@@ -319,7 +334,56 @@ def run_model_mixmasta(context, *args, **kwargs):
     # Final cleanup of temp directory
     shutil.rmtree(localpath)
 
-    response = {
-        "mixmaster_annotations": mm_ready_annotations
-    }
+    response = {"mixmaster_annotations": mm_ready_annotations}
     return response
+
+
+def scale_features(context, filename=None):
+    # 0 to 1 scaled dataframe
+    es = Elasticsearch(settings.ELASTICSEARCH_URL)
+    uuid = context["uuid"]
+    data_paths = context["dataset"]["data_paths"]
+    if not data_paths:
+        query = {
+            "query": {"match": {"id": uuid}},
+            "fields": ["data_paths"],
+            "_source": False,
+        }
+        es_response = es.search(index="indicators", body=query)
+        data_paths = es_response["hits"]["hits"][0]["fields"]["data_paths"]
+    paths_to_process = [
+        path for path in data_paths if "_str" not in path and "_normed" not in path
+    ]
+    data_paths = [path for path in data_paths if "_normed" not in path]
```

Comment:
I think this string should be `_normalized` instead of `_normed` based on the rest of the code, no?
But really, since we're storing the normalized files in a different field, I'm not sure we need this.